README.md

```diff
@@ -1,5 +1,5 @@
 [![Build Status](https://travis-ci.org/sul-dlss/common-accessioning.svg?branch=master)](https://travis-ci.org/sul-dlss/common-accessioning)
-[![Coverage Status](https://coveralls.io/repos/github/sul-dlss/common-accessioning/badge.svg?branch=master)](https://coveralls.io/github/sul-dlss/common-accessioning?branch=master)
+[![Code Climate](https://codeclimate.com/github/sul-dlss/common-accessioning/badges/gpa.svg)](https://codeclimate.com/github/sul-dlss/common-accessioning)
```

Comment:
This is the maintainability badge. The coverage badge will look like:

```markdown
[![Code Climate Test Coverage](https://codeclimate.com/github/sul-dlss/common-accessioning/badges/coverage.svg)](https://codeclimate.com/github/sul-dlss/common-accessioning/coverage)

```

Reply:
```suggestion
[![Code Climate Test Coverage](https://codeclimate.com/github/sul-dlss/common-accessioning/badges/coverage.svg)](https://codeclimate.com/github/sul-dlss/common-accessioning/coverage)
```